website/docs/r/compute_instance_ip.html.markdown

```diff
@@ -3,41 +3,44 @@ layout: "scaleway"
 page_title: "Scaleway: scaleway_compute_instance_ip"
 sidebar_current: "docs-scaleway-resource-compute-instance-ip"
 description: |-
-  Manages Scaleway compute instance ip.
+  Manages Scaleway Compute Instance IPs.
 ---
 
 # scaleway_compute_instance_ip
 
-Creates and manages Scaleway compute instance IPs.
+Creates and manages Scaleway Compute Instance IPs. For more information, see [the documentation](https://developers.scaleway.com/en/products/instance/api/#ips-268151).
 
 ## Example Usage
 
 ```hcl
-resource "scaleway_compute_instance_ip" "test" {}
+resource "scaleway_compute_instance_ip" "server_ip" {}
 ```
 
-## Argument Reference
+## Arguments Reference
 
 The following arguments are supported:
 
-* `project_id` - (Optional) The id of the project you want to attach this resource to.
-* `reverse` - (Optional) The reverse dns for this IP.
-* `zone` - (Optional) The zone you want to attach this resource to.
+- `reverse` - (Optional) The reverse DNS for this IP.
+- `server_id` - (Optional) The ID of the server you want to attach this resource to.
+- `project_id` - (Optional) The ID of the project you want to attach this resource to. If it is not provided, the provider `project_id` is used.
+- `zone` - (Optional) The [zone](https://developers.scaleway.com/en/quickstart/#zone-definition) you want to attach this resource to. If it is not provided, the provider `zone` is used.
 
 ## Attributes Reference
 
 The following attributes are exported:
 
-* `project_id` - The id of the project your resource is attached to.
-* `reverse` - The reverse dns for this IP.
-* `server_id` - The id of the server this resource is attached to.
-* `zone` - The zone your resource is attached to.
+- `id` - The ID of the IP.
+- `address` - The IP address.
+- `reverse` - The reverse DNS for this IP.
+- `server_id` - The ID of the server this resource is attached to.
+- `project_id` - The ID of the project you want to attach this resource to.
+- `zone` - The [zone](https://developers.scaleway.com/en/quickstart/#zone-definition) your resource is attached to.
```

Comment:
up

Reply:
I follow this order: https://github.com/terraform-providers/terraform-provider-scaleway/blob/cd5978fcd435d7b277c98decb93b0f26cf67b7c2/scaleway/resource_compute_instance_ip.go#L18-L36